src/resolve.cc

```diff
@@ -1,20 +1,50 @@
 #include "resolve.h"
 
 #include <iostream>
+#include <signal.h>
+#include <set>
+#include <vector>
+
+using std::set, std::string, std::cout, std::endl, std::vector, std::cerr;
```

Comment:
I personally don't like it. If our user like it, they can do it themselves.

Reply:
Oops, I forgot to delete them. I used them in my own impl, will fix soon